examples/bind_imported_app_to_incluster_operator_managed_PostgreSQL_db/README.md

```diff
@@ -0,0 +1,207 @@
+# Scenario - OpenShift Service Binding Operator - Binding an Imported app to an In-cluster Operator Managed PostgreSQL Database
```

Comment:
Let's just have this text "Binding an Imported app to an In-cluster Operator Managed PostgreSQL Database"

Reply:
Done - thanks!